tutorials/video_recorder.md

```diff
@@ -105,3 +105,10 @@ going to the world SDF file, locate the
 
 * **bitrate**: Video encoding bitrate in bps. This affects the quality of the
 generated video. The default bitrate is 2Mbps.
+  
+## Hardware-accelerated encoding
+
+Since Ignition Common 3.10.2, there is support for utilizing the power of GPUs
+to speed up the video encoding process. See the
+[Hardware-accelerated Video Encoding tutorial](https://github.com/ignitionrobotics/ign-common/blob/ign-common3/tutorials/install.md)
```

Comment:
The canonical location for our tutorials is the http://ignitionrobotics.org website, i.e. https://ignitionrobotics.org/api/common/3.10/tutorials.html. The GitHub markdown preview usually looks ok, but it doesn't render all the doxygen features properly (like cross-links, images, etc), so we shouldn't treat that preview as a reference.

We have nightly doc upload jobs, so the tutorial merged today in https://github.com/ignitionrobotics/ign-common/pull/169 should be live tomorrow. I believe its link will be https://ignitionrobotics.org/api/common/3.10/hw-encoding.html (I know the hardcoded minor version is annoying, we should eventually improve that (https://github.com/ignitionrobotics/docs/issues/52).

Reply:
Thanks, I fixed that link.